structure_tests/Makefile

```diff
@@ -19,22 +19,41 @@ VERSION_BUILD ?= 0
 
 VERSION ?= v$(VERSION_MAJOR).$(VERSION_MINOR).$(VERSION_BUILD)
 
+GOOS ?= $(shell go env GOOS)
+GOARCH = amd64
+ORG := github.com/GoogleCloudPlatform
+PROJECT := runtimes-common/structure_tests
+REPOPATH ?= $(ORG)/$(PROJECT)
+RELEASE_BUCKET ?= $(PROJECT)
```

Comment:
Don't we have another bucket for some of our other tools? It would be nice to consolidate these.

Reply:
ah whoops meant to change this. I don't think we have a consolidated one though, but I can make one